### PR TITLE
Add conventional changelog config docs

### PR DESCRIPTION
**Pull request by Yuuki-chan**

### CODE REVIEW

File: packages/conventional-changelog-config/README.md [hunk 1]
```diff
@@ -2,12 +2,115 @@
...
+`lerna publish --skip-git --conventional-commits --changelog-preset 
+@tophat/conventional-changelog-config --yes`
+
+Note: The `lerna` above should be the path to your Lerna executable if 
+Lerna is not installed globally.
+
+## Convention
+
+The changelog uses the commit message format from the 
+[Angular Convention](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular#commit-message-format).
```
Comment (sanchitgera) [inline packages/conventional-changelog-config/README.md]: Doesn't it use Top Hat's convention?

File: packages/conventional-changelog-config/README.md [hunk 2]
```diff
@@ -2,12 +2,115 @@
...
+
+### Included Commit Types
+
+The full list of commit types are listed [here](https://github.com/tophat/commit-utils/blob/master/packages/commit-utils-core/src/constants.js).
```
Comment (sanchitgera) [inline packages/conventional-changelog-config/README.md]: You can link back to the commit-utils README, the commit types are more readable there.

Comment (Yuuki-chan) [reply packages/conventional-changelog-config/README.md]: That's fair, will do!

File: packages/conventional-changelog-config/README.md (hunk 1 above)
Comment (Yuuki-chan) [reply packages/conventional-changelog-config/README.md]: Nope, we didn't come up with the format. We modified the commit types that we support but the format is still the same.

**Pull request merged**